module/VuFind/src/VuFind/Db/Entity/AccessToken.php

```diff
@@ -38,54 +38,71 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="access_token")
+ * @ORM\Table(name="access_token",
+ * indexes={@ORM\Index(name="user_id", columns={"user_id"})}
```

Comment:
I don't think this index needs to be annotated, because it's part of a foreign key relationship, which Doctrine should automatically recognize as indexed. However, please correct me if you've learned that I am incorrect about this! :-)